libraries/cloudharness-common/cloudharness/events/client.py

```diff
@@ -41,16 +41,29 @@ def create_topic(self):
                                         client_id=self.client_id)
         # ## Create topic
 
-        topic_list = [NewTopic(name=self.topic_id, num_partitions=1, replication_factor=1)]
+        topic_list = []
+        for topic in [NewTopic(name=self.topic_id, num_partitions=1, replication_factor=1)]:
+            try:
+                # validate if the Topic already exists
+                admin_client.create_topics(new_topics=[topic], validate_only=True)
+                # if not then add it to the topic_list
+                topic_list.append(topic)
+            except TopicAlreadyExistsError as e:
+                pass
+            except Exception as e:
+                log.error(f"Ups... We had an error creating the new Topic --> {e}")
```

Comment:
log.error better add parameter exc_info=True to get the stack trace

Reply:
done and updated also the other log error messages